src/models/session.ts

```diff
@@ -115,6 +115,10 @@ export class StaticSession {
   get weightMeasurement () {
     return this._sessionData.weightMeasurement
   }
+
+  get sessionPlanSequenceInstance () {
+    return this._sessionData.sessionPlanSequenceInstance
+  }
```

Comment:
This works fine for the `staticSession` model since all of that data is static and has no backing classes.